jobs/JEVS_MESOSCALE_STATS

```diff
@@ -88,11 +88,11 @@ if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
 elif [ $MODELNAME = sref ]; then
   export VDATE=${VDATE:-$PDYm2}
 
-  export COMIN=${COMIN:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT/$RUN
+  export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}/prep/$COMPONENT/$RUN
   export COMINsref=${COMINsref:-$(compath.py  sref/$sref_ver)}
   export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
   export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
-  export COMINsrefmean=${COMINsrefmean:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT
+  export EVSINsrefmean=${EVSINsrefmean:-$COMIN/prep/$COMPONENT}
```

Comment:
Since EVSINsrefmean looks good here in the J-job, please remove it from the drivers. 👍